astrokat/noisediode.py

```diff
@@ -83,42 +76,39 @@ def on(kat,
     else:
         # - use integer second boundary as that is most likely be an exact
         #   time that DMC can execute at
-        msg = ('Dry-run: Set all noise diodes with timestamp {} ({})'
-               .format(np.ceil(timestamp),
-                       time.ctime(timestamp)))
+        msg = "Dry-run: Set all noise diodes with timestamp" "{} ({})".format(
```

Comment:
The space between `timestamp` and `{}` got lost.

The black auto formatter also results in multiline strings sometimes ending up on one line again, but still in two parts.  This can be fixed as well, but just for this line.

`"Dry-run: Set all noise diodes with timestamp {} ({})".format(`